File: src/main/java/com/zuehlke/vr/openStreetmap/json/Track.java
package com.zuehlke.vr.openStreetmap.json;

public class Track {
    private int fromRef;
    private int toRef;

    public Track(int fromRef, int toRef) {
        this.fromRef = fromRef;
        this.toRef = toRef;
    }

    public int getFromRef() {
        return fromRef;
    }

    public void setFromRef(int fromRef) {
        this.fromRef = fromRef;
    }

    public int getToRef() {
        return toRef;
    }

    public void setToRef(int toRef) {
        this.toRef = toRef;
    }
}

File: unity/TrainTrip01/Assets/Scripts/Track.cs
using System.Collections;
using System.Collections.Generic;
using System;

public class Track
{
	public Node from;
	public Node to;
	
	public Track (JSONObject json, Dictionary<long, Node> nodes)
	{
		JSONObject fromObject = JSONHelper.getByKey (json, "from");
		JSONObject toObject = JSONHelper.getByKey (json, "to");
		
		long fromRef = JSONHelper.getByKey (fromObject, "@ref").i;
		long toRef = JSONHelper.getByKey (toObject, "@ref").i;
		
		from = nodes [fromRef];
		to = nodes [toRef];
	}
}